astrosat_users/templates/astrosat_users/email/customer_user_password_reset_key_message.txt

```diff
@@ -1,19 +0,0 @@
-{# used for resetting the password in the context of a user created by a customer manager #}
```

Comment:
Why has this been deleted, I get you have a new template for **invitations**, but that doesn't remove the need for users to forget their password at a later date and want to reset it.

Reply:
`allauth` already provides a template for that.  This template was purely for resetting a password in the context of a `CustomerUser` (not a `User`).